test/persist/mzcompose.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.mzcompose import Composition
+from materialize.mzcompose.services import Service
+
+SERVICES = [
+    Service(
+        "maelstrom-persist",
+        {"mzbuild": "maelstrom-persist", "volumes": ["./maelstrom:/store"]},
+    ),
+]
+
+
+def workflow_default(c: Composition) -> None:
```

Comment:
this is needlessly hardcoded, but I'm mostly just looking to get this hooked up to CI without spending too much time on it. for some reason qemu seems to be more unhappy about running things locally than the last time I worked on this, so iteration is hard